### PR TITLE
Update labels.json

### PR DESCRIPTION
**Pull request by JSimian**
Add Some PT-BR translarions

### CODE REVIEW

File: labels.json [hunk 1]
```diff
@@ -1956,6 +1961,11 @@
                 "lang": "en",
                 "name": "Peafowl",
                 "description": "This user is a Peafowl! [Category: Avian/Bird]"
+            },
+            {
+                "lang": "pt-BR",
+                "name": "Pavão",
+                "description": "Este usuário é um Pavão! AKA: Peafowl"
```
Comment (astrabun) [inline labels.json]: Please include: `[Category: Ave/Pássaro]`

File: labels.json [hunk 2]
```diff
@@ -1996,6 +2021,11 @@
                 "lang": "en",
                 "name": "Mightyena",
                 "description": "This user is a Mightyena! [Category: Pokemon]"
+            },
+            {
+                "lang": "pt-BR",
+                "name": "Mightyena",
+                "description": "Este usuário é um Mightyena! [Categoria: Pokemon]  AKA: Mightyena"
```
Comment (astrabun) [inline labels.json]: Please keep as `Category` as this is parsed out on https://sonasky-browse.bunnys.ky/ (I need to update the README to include this request, that's my bad)

File: labels.json [hunk 3]
```diff
@@ -2026,6 +2066,11 @@
                 "lang": "en",
                 "name": "Macaw",
                 "description": "This user is a Macaw! [Category: Avian/Bird]"
+            },
+            {
+                "lang": "pt-BR",
+                "name": "Arara",
+                "description": "Este usuário é uma Arara!  AKA: Macaw"
```
Comment (astrabun) [inline labels.json]: `[Category: Ave/Pássaro]`

File: labels.json [hunk 4]
```diff
@@ -2196,6 +2321,11 @@
                 "lang": "en",
                 "name": "Badger",
                 "description": "This user is a Badger! [Category: Mustelid]"
+            },
+            {
+                "lang": "pt-BR",
+                "name": "Texugo",
+                "description": "Este usuário é um Texugo!  AKA: Badger"
```
Comment (astrabun) [inline labels.json]: If possible, please include  `[Category: Mustelid]` (or if there is a translation of Mustelid, then `[Category: translation_here]`

**Review by astrabun — CHANGES_REQUESTED** [review]
Thanks so much :) Left a few comments if you don't mind addressing those before merging that would be awesome

**Comment by JSimian** [comment]
Ah! No problem, sorry for breaking some standards, it went unnoticed.
I'll be more careful

**Comment by astrabun** [comment]
> Ah! No problem, sorry for breaking some standards, it went unnoticed. I'll be more careful

No worries at all! I only added that to the readme within the last hour, not your fault!

This looks great now, merging!

**Pull request merged**